### ISSUE TITLE
GameGuru MAX - Suggestion - Time to Split Storyboard?

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4116]
With all the new stuff going on with the Storyboard editor, it might be time to split it into two parts. Part 1 would be the Storyboard as we know it and would be for levels/maps, about screens, title screens, splash screens, etc. Basically the stuff that's been there since it's inception. Part 2 would be for all the stuff that's not part of that flow, such as the HUD, Inventory screens, Map screens, etc.

The idea here is that the first Storyboard would be for game flow - Splash screen to loading to playing the game to end screens (winning, etc.). The second would be for stuff that's not really a part of that flow, but used intermittently throughout game play. This is stuff that does not need to be "hooked up" to other nodes in the Storyboard editor.

The main reason I am suggesting this is the Storyboard could become very busy and crowded soon with all the things we could add there. By having two Storyboard areas, then we could dedicate one to the "flow" of the game and the other to reused elements, such as all the HUDs used throughout the game. 

Perhaps how this could be done is by having a tab at the top of the Storyboard editor to allow the end-user to access which part they want to work on - Storyboard or HUDs, for example. 

Just a thought.

**Comment by Necrym59** [comment]
Yes i think it may need to happen to segregate the huds from the storyboard if there is going to be a range of huds cluttering up the storyboard
